src/App.vue

```diff
@@ -135,6 +135,7 @@ export default {
 	},
 
 	beforeMount() {
+		console.debug('Loading theming data for notification bell styling')
```

Comment:
Could you use `@nextcloud/logger`? :)

Reply:
Something is preventing the app to continue. I'd like to avoid loading more dependencies just to find out what and then revert it 😬